README.md

```diff
@@ -147,6 +147,82 @@ This project was made as Go learning opportunity. The initial release was create
 	@reboot gitlab-ci-multi-runner run &>gitlab-ci-multi-runner.log
 	```
 
+### Docker image installation and configuration (run gitlab-ci-multi-runner as a container)
+
+1. Pull the image (optional):
+  ```bash
+  $ docker pull ayufan/gitlab-ci-multi-runner:latest
+  ```
+
+1. Start the container:
+  ```bash
+  $ docker run -d --name multi-runner --restart always \
+      ayufan/gitlab-ci-multi-runner:latest
+  ```
+
+  If you are planning on using Docker as the method of spawing runners you'll need to mount your docker socket like so:
+  ```bash
+  $ docker run -d --name multi-runner --restart always \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      ayufan/gitlab-ci-multi-runner:latest
+  ```
+
+  Additionally, you can mount your own custom data folder to be used for loading the config and any other resources:
+  ```bash
+  $ docker run -d --name multi-runner --restart always \
+      -v /PATH/TO/DATA/FOLDER:/data \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      ayufan/gitlab-ci-multi-runner:latest
+  ```
+
+  OR you can use a data container to mount you custom data folder:
+  ```bash
+  $ docker run -d --name multi-runner-data -v /data \
+      busybox:latest /bin/true
+
+  $ docker run -d --name multi-runner --restart always \
+      --volumes-from multi-runner-data \
+      -v /var/run/docker.sock:/var/run/docker.sock \
+      ayufan/gitlab-ci-multi-runner:latest
+  ```
+
+1. Setup the runner:
+  ```bash
+  $ docker exec -it multi-runner gitlab-ci-multi-runner setup
+  Please enter the gitlab-ci coordinator URL (e.g. http://gitlab-ci.org:3000/ )
+  https://ci.gitlab.org/
+  Please enter the gitlab-ci token for this runner
+  xxx
+  Please enter the gitlab-ci description for this runner
+  my-runner
+  INFO[0034] fcf5c619 Registering runner... succeeded
+  Please enter the executor: shell, docker, docker-ssh, ssh?
+  docker
+  Please enter the Docker image (eg. ruby:2.1):
+  ruby:2.1
+  INFO[0037] Runner registered successfully. Feel free to start it, but if it's running already the config should be automatically reloaded!
+  ```
+
+1. Runner should be started already and you are ready to build your projects!
+
+#### Update
+
+1. Pull the latest version:
+  ```bash
+  $ docker pull ayufan/gitlab-ci-multi-runner:latest
+  ```
+
+1. Stop and remove the existing container:
+  ```bash
+  $ docker stop multi-runner && docker rm multi-runner
```

Comment:
Wouldn't it be better to send a `SIGTERM` (via `docker stop container`) and allow the process to clean up and exit versus sending a `SIGKILL` (via `docker rm -f container`) and halting the process?